python-lib/language_support.py

```diff
@@ -84,3 +84,47 @@
 
 Models with Creative Commons licenses are not included because this plugin is licensed under Apache-2
 """
+
+SPACY_LANGUAGE_MODELS_LEMMATIZATION = ["en", "es", "nb", "fr", "de", "ru", "pl"]
+"""list: Languages that have a SpaCy pre-trained model with a Lemmatizer component.
+When using a pre-trained pipeline to lemmatize, you need to have in your SpaCy Language pipeline:
+-either SpaCy 'morphologizer' + 'lemmatizer'
+-or SpaCy 'tagger' + 'attribute ruler' + 'lemmatizer'
+depending on the pre-trained pipeline built-in components"""
+
+SPACY_LANGUAGE_MODELS_MORPHOLOGIZER = ["es", "nb", "fr", "de", "ru", "pl"]
+"""list: Languages that have a SpaCy pre-trained model with a Morphologizer component."""
+
+SPACY_LANGUAGE_LOOKUP = [
+    "ca",
+    "cs",
+    "da",
+    "hr",
+    "hu",
+    "id",
+    "it",
+    "lb",
+    "lt",
+    "pt",
+    "ro",
+    "sr",
+    "tl",
+    "tr",
+    "ur",
+    "en",
+    "de",
+    "es",
+    "nb",
+    "fr",
+    "mk",
+    "nl",
+    "sv",
+]
+"""list: Languages that have available SpaCy lookup tables for lemmatization. 
+The lookup tables are available at https://github.com/explosion/spacy-lookups-data/tree/master/spacy_lookups_data/data
+"""
+
+SPACY_LANGUAGE_RULES = ["bn", "el", "fa"]
+"""list: Languages that have available SpaCy rule tables for lemmatization
```

Comment:
How are rules tables different from lookup tables ?

Reply:
From spacy documentation https://spacy.io/usage/linguistic-features#lemmatizer-lookup: 

About the lookup tables:
> The lookup lemmatizer looks up the token surface form in the lookup table without reference to the token’s part-of-speech or context.

About the rule tables:
> When training pipelines that include a component that assigns part-of-speech tags (a morphologizer or a tagger with a POS mapping), a rule-based lemmatizer can be added using rule tables from spacy-lookups-data